addons/payment/static/src/js/checkout_form.js

```diff
@@ -19,6 +19,7 @@
          * @constructor
          */
         init: function () {
+            this.txContext = {}; // It is important to initialize it before `start`
```

Comment:
This should also be done in  `manage_form.js` because it also uses the mixin. But I would do this
```diff
diff --git a/addons/payment/static/src/js/checkout_form.js b/addons/payment/static/src/js/checkout_form.js
index 51b17b4777be..6d8d194a7273 100644
--- a/addons/payment/static/src/js/checkout_form.js
+++ b/addons/payment/static/src/js/checkout_form.js
@@ -19,6 +19,7 @@
          * @constructor
          */
         init: function () {
+            paymentFormMixin.init.call(this);
             this.formType = 'checkout';
             const preventDoubleClick = handlerMethod => {
                 return debounce(handlerMethod, 500, true);
diff --git a/addons/payment/static/src/js/manage_form.js b/addons/payment/static/src/js/manage_form.js
index 9544bf30da1d..76d810c00ee3 100644
--- a/addons/payment/static/src/js/manage_form.js
+++ b/addons/payment/static/src/js/manage_form.js
@@ -24,6 +24,7 @@ import { debounce } from "@web/core/utils/timing";
          * @constructor
          */
         init: function () {
+            paymentFormMixin.init.call(this);
             this.formType = 'manage';
             const preventDoubleClick = handlerMethod => {
                 return debounce(handlerMethod, 500, true);
diff --git a/addons/payment/static/src/js/payment_form_mixin.js b/addons/payment/static/src/js/payment_form_mixin.js
index fae686c1aa93..15f113eab7cc 100644
--- a/addons/payment/static/src/js/payment_form_mixin.js
+++ b/addons/payment/static/src/js/payment_form_mixin.js
@@ -8,6 +8,10 @@
 
     export default {
 
+        init: function () {
+            this.txContext = {};
+        },
+
         /**
          * @override
          */

```